### PR TITLE
display information msg when submit or copy

### PR DESCRIPTION
**Pull request by cmangeat**

### CODE REVIEW

File: c2cgeoform/scaffolds/c2cgeoform/acceptance_tests/excavation_test.py_tmpl [hunk 1]
```diff
@@ -254,6 +260,8 @@ class TestExcavationViews():
 
         situation_ids = [situation.id for situation in exc.situations]
         assert set([3, 4, 5]) == set(situation_ids)
+        assert 'your submission has been token into account.' == \
```
Comment (arnaud-morvan) [inline c2cgeoform/scaffolds/c2cgeoform/acceptance_tests/excavation_test.py_tmpl]: **Y**our submission has been t**a**ken into account

File: c2cgeoform/scaffolds/c2cgeoform/acceptance_tests/excavation_test.py_tmpl [hunk 2]
```diff
@@ -303,6 +311,10 @@ class TestExcavationViews():
         assert '' == form['permission_id'].value
         assert exc.contact_persons[0].permission_id != form['permission_id'].value
 
+        assert 'please check that the copy fits before submitting.' == \
```
Comment (arnaud-morvan) [inline c2cgeoform/scaffolds/c2cgeoform/acceptance_tests/excavation_test.py_tmpl]: **P**lease

File: c2cgeoform/scaffolds/c2cgeoform/acceptance_tests/excavation_test.py_tmpl [hunk 3]
```diff
@@ -180,6 +182,8 @@ class TestExcavationViews():
         situation_ids = [situation.id for situation
                           in exc.situations]
         assert set([3, 4, 5]) == set(situation_ids)
+        assert 'your submission has been token into account.' == \
```
Comment (arnaud-morvan) [inline c2cgeoform/scaffolds/c2cgeoform/acceptance_tests/excavation_test.py_tmpl]: **Y**our submission has been t**a**ken into account

File: c2cgeoform/views/abstract_views.py [hunk 4]
```diff
@@ -336,9 +340,17 @@ def edit(self):
...
+            rendered = form.render(
+                dict_,
+                request=self._request,
+                actions=self._item_actions(obj),
+                msg_col=[MSG_COL[self._request.params['msg_col']]])
+        else:
+            rendered = form.render(
+                dict_,
+                request=self._request,
+                actions=self._item_actions(obj))
```
Comment (arnaud-morvan) [inline c2cgeoform/views/abstract_views.py]: Is it possible to create a variable `msg_col` instead of duplicating the call to form.render.

Comment (cmangeat) [reply c2cgeoform/views/abstract_views.py]: I try to, but when setting 'msg_col = None', exists at chameleon side is true !

**Pull request merged**